changedetectionio/tests/test_import.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+import time
+
+from flask import url_for
+
+from .util import live_server_setup
+
+
+def test_import(client, live_server):
+
+    live_server_setup(live_server)
+
+    # Give the endpoint time to spin up
+    time.sleep(1)
+
+    res = client.post(
+        url_for("import_page"),
+        data={
+            "urls": """https://example.com
+https://example.com tag1
+https://example.com tag1, other tag"""
```

Comment:
hmm, one thing, please change these to different URLs, i see there is a bug here in that it is not de-duping ! well spotted :dancers: 

```
https://example1.com
https://example2.com tag1
https://example3.com tag1, other tag
```

Reply:
Haha, I thought about that and actually first used those URLs - however, it is making those requests in live, and I believe `https://example2.com` made the test take quite long until that request timed out.
The clone functionality allows for the same URL to exist multiple times, therefore I didn't think deduplication is a problem, and allowing the same URL under different tags (and presumably different settings) is a feature, and not a bug.

Reply:
> https://example2.com made the test take quite long until that request timed out.

Reply:
> > https://example2.com made the test take quite long until that request timed out.

Reply:
>  https://example2.com made the test take quite long until that request timed out.

Reply:
Hmm interesting> https://example2.com made the test take quite long until that request timed out.

Reply:
hmm I'll have to check that out, all requests go into a background queue runner.. standby

 > https://example2.com made the test take quite long until that request timed out.